bitsv/network/services/network.py

```diff
@@ -74,54 +90,34 @@ def get_transaction(cls, txid):
         :rtype: ``Transaction``
         """
 
-        for api_call in cls.GET_TX_MAIN:
+        for api_call in self.GET_TRANSACTION:
             try:
                 return api_call(txid)
-            except cls.IGNORED_ERRORS:
+            except self.IGNORED_ERRORS:
                 pass
 
         raise ConnectionError('All APIs are unreachable.')
 
-    @classmethod
-    def get_tx_amount(cls, txid, txindex):
-        """Gets the amount of a given transaction output.
-
-        :param txid: The transaction id in question.
-        :type txid: ``str``
-        :param txindex: The transaction index in question.
-        :type txindex: ``int``
-        :raises ConnectionError: If all API services fail.
-        :rtype: ``Decimal``
-        """
-
-        for api_call in cls.GET_TX_AMOUNT_MAIN:
-            try:
-                return api_call(txid, txindex)
-            except cls.IGNORED_ERRORS:
-                pass
-
-        raise ConnectionError('All APIs are unreachable.')
-
-    @classmethod
-    def get_unspent(cls, address):
+    def get_unspents(self, address, sort=None):
         """Gets all unspent transaction outputs belonging to an address.
 
         :param address: The address in question.
         :type address: ``str``
         :raises ConnectionError: If all API services fail.
-        :rtype: ``list`` of :class:`~bitcash.network.meta.Unspent`
+        :param address: Address to get utxos for
+        :param sort: 'value:desc' or 'value:asc' to sort unspents by descending/ascending order respectively
```

Comment:
Were you still planning to offer a nicer syntax for this, e.g. splitting it into `sort` and `sort_direction` (with default `desc`)?

Reply:
done in #25